src/org-unit-tree/OrgUnitTree.component.js

```diff
@@ -234,6 +244,14 @@ class OrgUnitTree extends React.Component {
             this.props.onChangeCurrentRoot(currentOu);
         };
 
+        const handleTypeLabel = () => {
+            if (selectableLevel !== undefined) {
+                return "radio";
```

Comment:
I forgot about this: whether a radio or checkbox is to be used does not depend on `selectableLevel`, it should be a separate prop (if we really want to use a radio input here).

Reply:
In the last commit, you can see another prop "typeInput". By default, without this prop, it would be checkbox.